src/openrct2-ui/windows/RideList.cpp

```diff
@@ -708,7 +708,9 @@ static void window_ride_list_scrollpaint(rct_window* w, rct_drawpixelinfo* dpi,
                 formatSecondary = STR_QUEUE_EMPTY;
                 {
                     uint16_t arg;
-                    std::memcpy(&arg, gCommonFormatArgs + 2, sizeof(uint16_t));
+                    ft.Rewind();
```

Comment:
This code is ridiculous. First we get the total queue length and put it in the buffer. Then we rewind the buffer then we increment the buffer to the total queue length. Then we copy out the total queue length. How about we just use the total queue length directly!